tests/arns.test.mjs

```diff
@@ -133,6 +133,8 @@ describe('ArNS', async () => {
     };
   };
 
+  const basePermabuyPrice = 2500000000;
```

Comment:
Thanks for this. Wouldn't hate some underscores to separate the 3-digit clusters.

Reply:
bb1deb5